### ISSUE TITLE
--cd error with example images

### ISSUE THREAD

**Issue by j-ogas** [hpc/charliecloud#108]
First, it is possible I am using the `--cd` command incorrectly. The usage information provided by --help is somewhat ambiguous and I am unable (with a quick glance) to find information about it in the documentation (https://github.com/hpc/charliecloud.git).

Using the example images provided by the test suite (centos6 and ubuntu1604):

**Expected Behavior**

`ch-run --cd /etc /var/tmp/images/centos6 -- /bin/bash`

Enter an interactive container in /etc.

**Actual Behavior**

`ch-run --cd /etc /var/tmp/images/centos6 -- /bin/bash`
`ch-run: can't execve(2) user command: No such file or directory (ch-run.c:493 2)`

**Notes**

- I get the same error when using the same command with ubuntu1604 image

- I get the same error when targeting other valid directories e.g., `--cd /bin`, `--cd /var`, `--cd /home`, etc.

- The command works if I target the root dir e.g., `ch-run --cd / /var/tmp/images/centos6 -- /bin/bash` (but seemingly defeats the point)

**Comment by reidpr** [comment]
You're using `--cd` correctly.

However, I can't reproduce the problem:

```
$ ch-run --cd /etc $CH_TEST_IMGDIR/centos6 -- pwd
/etc
$ ch-run --cd /etc $CH_TEST_IMGDIR/centos6 -- /bin/bash -c 'pwd'
/etc
$ ch-run --cd /etc $CH_TEST_IMGDIR/ubuntu1604 -- /bin/bash -c 'pwd'
/etc
```

* Are you sure your images are in order?
* Does the test suite pass? (We do have a `--cd` test in there; do those commands work?)
* Does it work with the `chtest` image?

**Comment by j-ogas** [comment]
It was a user error. It works as expected. Apologies for the inconvenience.

**Comment by reidpr** [comment]
What was the user error? Maybe the documentation or UX needs to be improved.

**Comment by j-ogas** [comment]
I was afraid (but suspected) you might ask. I brought this shame upon myself, so I will own it (lol):

This user error is the result of me typing `bin/bash` instead of `/bin/bash` and thinking that I typed the latter. Yikes. I should have copy/paste from the terminal instead of re-type what I thought I did. 

It was easy for me to think I was simply using the command wrong, hence my visit to the docs and `--help`. The documentation is fine, the user needs improvement ;P
